nativelib/src/main/resources/scala-native/gc/commix/Phase.c

```diff
@@ -9,15 +9,34 @@
 #include <errno.h>
 #include <stdlib.h>
 
+/*
+If in OSX, sem_open cannot create a semaphore whose name is longer than
+PSEMNAMLEN characters. Else, we stick to POSIX compliance:
+
+  For POSIX.1-2017, XSI-conforming implementations shall set the
+  symbolic constant _XOPEN_UNIX to a value other than -1 and shall set the
+  symbolic constant _XOPEN_VERSION to the value 700.
```

Comment:
This is defined here: https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap02.html